ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -687,6 +687,7 @@ OA:
   - outside_air_relative_humidity_sensor
   - outside_air_dewpoint_temperature_sensor
   - outside_air_wetbulb_temperature_sensor
+  - outside_air_wetbulb_temperature_setpoint
```

Comment:
We cannot control outside air wetbulb temperature. What is this controlling? Leaving wetbulb temp?

Reply:
This is for monitoring and at the same time it is used to change cooling tower setpoint

Reply:
changed to condensing_supply_water_temperature_setpoint